src/server/services/wfs3/qgswfs3handlers.cpp

```diff
@@ -1077,7 +1077,8 @@ void QgsWfs3CollectionsItemsHandler::handleRequest( const QgsServerApiContext &c
     QgsJsonExporter exporter { mapLayer };
     exporter.setAttributes( featureRequest.subsetOfAttributes() );
     exporter.setAttributeDisplayName( true );
-    exporter.setSourceCrs( mapLayer->crs() );
+    // Trick! See: https://github.com/qgis/QGIS/commit/7d2000bc30b2160692baf43c22d7ef4b47e52cff
+    exporter.setSourceCrs( QgsCoordinateReferenceSystem( 4326, QgsCoordinateReferenceSystem::EpsgCrsId ) );
```

Comment:
I don't understand this -- are you trying to disable transformation? if so, just setting the source crs to an invalid crs should work.

Reply:
Yes, that was the intent.

Reply:
In any event, this is a transitory solution, until this is backported: https://github.com/qgis/QGIS/pull/32386#issuecomment-549176578 and I can use an explicit call to `exporter.setTransformGeometries( false );`